docs/manuel/src/00-netcontrol/nftables.md

```diff
@@ -71,6 +71,24 @@ Et elle :
 
 `ether saddr map @netcontrol-mac2mark` signifie que la mark est récupérée depuis l'entrée dans la map correspondant à `ether saddr`, ou la MAC de la source.
 
+### Bypass
+
+```bash
+nft add chain insalan netcontrol-debypass { type filter hook prerouting priority 0; }
+
+nft add rule insalan netcontrol-debypass meta mark > 1024 meta mark set meta mark ^ 1024
+```
+
+Cette règle s'applique aux paquets qui:
+- `meta mark > 1024` : Ont le bypass activé.
+
+Et elle :
+- `meta mark set meta mark ^ 1024` : Leur enlève le bypass, tout simplement (l'opérateur bitwise ^ va simplement toggle le bit de 1024, c'est pour ça qu'on n'applique la règle qu'aux marks supérieures à 1024) (on suppose ne jamais avoir de mark supérieure à 2048, ça n'aurait pas de sens de toute façon).
```

Comment:
Si tu veux _enlever_ le 10e bit, il ne faut pas XORer avec 1024 mais ANDer avec le NOT de 1024.